HelloWorld/main.cpp

```diff
@@ -0,0 +1,28 @@
+#include <iostream>  
+using namespace std;  
+
+//Source: https://www.javatpoint.com/prime-number-program-in-cpp
+
+int isPrime(int n)
```

Comment:
When possible, prefer static methods in implementation files:
```suggestion
static int isPrime(int n)
```

Reply:
Actually, this should probably return `bool` too.